src/rt/sections_ldc.d

```diff
@@ -96,22 +96,28 @@ private
             Section(SEG_DATA, SECT_COMMON)
         ];
     }
-    else version (CRuntime_Microsoft)
+    version (MinGW)
     {
-        extern extern (C) __gshared
+        extern (C) __gshared
```

Comment:
was dropping the first `extern` intended?

Reply:
Oh, good catch! That's definitely wrong, as the symbols are left initialized with nulls, and the GC doesn't have a single global as GC root.

Reply:
More context: these symbols are to be provided by the linker. An extra underscore is prepended by LDC automatically.